src/components/LoansYouMightLike/lymlContainer.vue

```diff
@@ -257,6 +313,33 @@ export default {
 					fetchPolicy: 'network-only',
 				});
 			}
+		},
+		setupExperimentState() {
+			// read the experiment version from the client cache
+			const localExperiment = this.apollo.readFragment({
+				id: 'Experiment:ml_loan_to_loan',
+				fragment: experimentVersionFragment,
+			}) || {};
+
+			if (localExperiment.version === 'control' || localExperiment.version === 'shown') {
+				this.expMlLoanToLoan = localExperiment.version === 'shown';
+			}
+		},
+		fireExperimentTracking() {
+			// track loans shown
```

Comment:
Will remove these console logs after I've had some additional testing in dev

Reply:
cool totally understand the interest in testing this where that query can run properly